CMakeLists.txt

```
@@ -2096,7 +2107,12 @@ target_link_libraries(mixxx-lib PRIVATE mixxx-proto)
 target_include_directories(mixxx-lib SYSTEM PUBLIC lib/rigtorp/SPSCQueue/include)
 
 # Qt
-find_package(QT NAMES Qt5 COMPONENTS Core REQUIRED)
+if(QT6)
+  find_package(QT NAMES Qt6 COMPONENTS Core REQUIRED)
+  set(QT6_NEW_COMPONENTS "SvgWidgets;Core5Compat")
```

Maybe try something like this? 

```suggestion
find_package(QT NAMES Qt6 COMPONENTS Core SvgWidgets Core5Compat REQUIRED)
target_link_libraries(mixxx-lib PUBLIC Qt6::SvgWidgets Qt6::Core5Compat)
```

---

I don't think that will work because this does not link the QT (capitals) package; only the Qt[5/6] package. Also I think it would be easy to overlook having this apart from the rest of the components searched for.